src/Users.js

```diff
@@ -253,13 +254,23 @@ class Users extends React.Component {
     }
   }
 
+  fetchLoans = props => {
+    const {
+      reset,
+      GET,
+    } = props.mutator.loans;
+
+    reset();
```

Comment:
I would also add some mechanism to prevent loans fetching when there is one already in progress.